tools/PublishModules.ps1

```diff
@@ -174,10 +179,20 @@ function Change-RMModule
 		  Write-Output "Expanding $zipPath"
 		  Expand-Archive $zipPath
 		  Write-Output "Adding PSM1 dependency to $unzippedManifest"
-		  Add-PSM1Dependency -Path $unzippedManifest
-		  Write-Output "Removing module manifest dependencies for $unzippedManifest"
-		  Remove-ModuleDependencies -Path $unzippedManifest
 
+          if ($moduleName -ne "AzureStack")
```

Comment:
It isn't run against AzureRM, it would only need to be run for a module that contained dlls, not a wrapper module, which only included other modules